migration/src/main/resources/META-INF/rewrite/axon-jakarta.yml

```diff
@@ -0,0 +1,74 @@
+type: specs.openrewrite.org/v1beta/recipe
+name: org.axonframework.axon.UpgradeAxonFramework_4_7_Jakarta
+displayName: Upgrade to Axonframework 4.7 Jakarta
+description: This is what you will want if you also upgrade to Spring 6, Spring Boot 3, or another framework that depends on jakarta packages.
+recipeList:
+  # Convert to Spring Boot 3, Spring FrameWork 6, Jakarta packages, JUnit 5, Java 17, etc.
+  #TODO - org.openrewrite.java.spring.boot3.UpgradeSpringBoot_3_0
+  # Swap 4.6 dependencies for 4.7 dependencies
+  - org.openrewrite.maven.ChangeDependencyGroupIdAndArtifactId:
+      oldGroupId: org.axonframework
+      oldArtifactId: axon-configuration-jakarta
+      newArtifactId: axon-configuration
+      newVersion: 4.7.x
+  - org.openrewrite.maven.ChangeDependencyGroupIdAndArtifactId:
+      oldGroupId: org.axonframework
+      oldArtifactId: axon-eventsourcing-jakarta
+      newArtifactId: axon-eventsourcing
+      newVersion: 4.7.x
+  - org.openrewrite.maven.ChangeDependencyGroupIdAndArtifactId:
+      oldGroupId: org.axonframework
+      oldArtifactId: axon-messaging-jakarta
+      newArtifactId: axon-messaging
+      newVersion: 4.7.x
+  - org.openrewrite.maven.ChangeDependencyGroupIdAndArtifactId:
+      oldGroupId: org.axonframework
+      oldArtifactId: axon-modelling-jakarta
+      newArtifactId: axon-modelling
+      newVersion: 4.7.x
+  # Convert legacy JPA packages to Jakarta JPA packages
```

Comment:
I would expect the `org.axonframework.common.jpa.PagingJpaQueryIterable` to `org.axonframework.common.legacyjpa.PagingJpaQueryIterable` too here.

Reply:
Did you mean the other way around? As this is the migration _to_ Jakarta, I've added
```yaml
  - org.openrewrite.java.ChangePackage:
      oldPackageName: org.axonframework.common.legacyjpa
      newPackageName: org.axonframework.common.jpa
```